media-video/motion/motion-4.1.1-r2.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit autotools readme.gentoo-r1 systemd user
+
+DESCRIPTION="A software motion detector"
+HOMEPAGE="https://motion-project.github.io"
+SRC_URI="https://github.com/Motion-Project/${PN}/archive/release-${PV}.tar.gz -> ${PN}-release-${PV}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc64 ~sparc ~x86"
+IUSE="ffmpeg libav mariadb mmal mysql postgres sqlite v4l webp"
+
+RDEPEND="
+	virtual/jpeg:=
+	ffmpeg? (
+		libav? ( media-video/libav:= )
+		!libav? ( media-video/ffmpeg:0= )
+	)
+	mmal? ( media-libs/raspberrypi-userland )
+	mysql? (
```

Comment:
I'm sorry but I've gotta block this until https://bugs.gentoo.org/667918 is resolved. Apparently the semantic of the flag is unclear at the moment, and you've chosen the less popular variant.